infra/gcp/ensure-prod-storage.sh

```diff
@@ -358,6 +358,39 @@ color 6 "Handling special cases"
         "k8s-infra-prow-build-trusted.svc.id.goog[test-pods/k8s-infra-gcr-vuln-scanning]" \
         "${PROD_PROJECT}" \
         $(svc_acct_email "${PROD_PROJECT}" "${PROMOTER_VULN_SCANNING_SVCACCT}")
+
+    color 6 "Ensuring prod promoter vuln dashboard svcacct exists"
+    ensure_service_account \
+        "${PROD_PROJECT}" \
+        "${VULN_DASHBOARD_SVCACCT}" \
+        "k8s-infra container image vuln scanning"
+
+    color 6 "Empowering promoter-scanning namespace to use prod promoter vuln dashboard svcacct"
+    empower_ksa_to_svcacct \
+        "k8s-prow-builds.svc.id.goog[test-pods/k8s-infra-gcr-vuln-dashboard]" \
+        "${PROD_PROJECT}" \
+        $(svc_acct_email "${PROD_PROJECT}" "${VULN_DASHBOARD_SVCACCT}")
+    empower_ksa_to_svcacct \
+        "k8s-infra-prow-build-trusted.svc.id.goog[test-pods/k8s-infra-gcr-vuln-dashboard]" \
```

Comment:
Can you add this service account to https://github.com/kubernetes/k8s.io/blob/master/infra/gcp/clusters/projects/k8s-infra-prow-build-trusted/prow-build-trusted/resources/build-serviceaccounts.yaml